### PR TITLE
Added Yeswehack scope 

### PR DESCRIPTION
**Pull request by bbacountb**
Added Yeswehack scope, with good json syntax

### CODE REVIEW

**Comment by GeorginaReeder** [comment]
Thanks for your contribution @bbacountb !

**Comment by bbacountb** [comment]
Can this be reviewed?

**Comment by jdouliez** [comment]
Waiting this PR !!!! :)

File: chaos-bugbounty-list.json [hunk 1]
```diff
@@ -4383,6 +4391,14 @@
         "grammarly.net"
       ]
     },
+    {
+      "name": "Govtech",
```
Comment (dwisiswant0) [inline chaos-bugbounty-list.json:4395]:
```suggestion
      "name": "GovTech",
```

File: chaos-bugbounty-list.json [hunk 2]
```diff
@@ -4508,6 +4524,26 @@
         "hboeck.de"
       ]
     },
+    {
+      "name": "Harman International",
```
Comment (dwisiswant0) [inline chaos-bugbounty-list.json:4528]:
```suggestion
      "name": "HARMAN International",
```

File: chaos-bugbounty-list.json [hunk 3]
```diff
@@ -7963,6 +8009,14 @@
         "s-cloud.net"
       ]
     },
+    {
+      "name": "Spacelift Io",
```
Comment (dwisiswant0) [inline chaos-bugbounty-list.json:8013]:
```suggestion
      "name": "Spacelift.io",
```

File: chaos-bugbounty-list.json [hunk 4]
```diff
@@ -9244,6 +9301,17 @@
         "vertexprotocol.com"
       ]
     },
+    {
+      "name": "Vfs",
```
Comment (dwisiswant0) [inline chaos-bugbounty-list.json:9305]:
```suggestion
      "name": "VFS Global",
```

**Review by dwisiswant0 — CHANGES_REQUESTED** [review]

**Review by ehsandeep — APPROVED** [review]
@dwisiswant0 we can address workflow related issues separately - https://github.com/projectdiscovery/public-bugbounty-programs/issues/887

**Pull request merged**